api/migrations/0088_auto_20201105_0943.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 2.2.13 on 2020-11-05 09:43
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('api', '0087_auto_20200918_0922'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='country',
+            name='fdrs',
+            field=models.CharField(max_length=6, null=True, verbose_name='FDRS'),
+        ),
+        # migrations.AlterField(
```

Comment:
@GregoryHorvath @batpad This creeped up when I generated migrations but I've commented out. Let me know if I should keep this.

Reply:
There was a discrepancy in the `verbose_name` only I fixedsome time ago. I guess I forgot to add a migration for that. You're free to uncomment that please.